.pipelines/clean-subscription.yml

```diff
@@ -9,7 +9,7 @@ parameters:
 resources:
   containers:
     - container: golang
-      image: registry.access.redhat.com/ubi8/go-toolset:1.22.9-2
+      image: registry.access.redhat.com/golang-builder--partner-share:rhel-9-golang-1.22-openshift-4.19
```

Comment:
@cadenmarchese obviously no such image exists - can this pipeline use the service connection in ADO to pull from `arointsvc`?

Reply:
From my docs dive, it looks like it will "just work" if the service connection in ADO allows access from this pipeline.